Makefile

```diff
@@ -201,6 +208,25 @@ golangci-lint: generate-embeddedyamls
 
 unit: generate-embeddedyamls
 
+# Operator SDK
+# On version bumps, the checksum will need to be updated manually.
+# If necessary, the verification *keys* can be updated as follows:
+# * update scripts/operator-sdk-signing-key.asc, import the relevant key,
+#   and export it with
+#     gpg --armor --export-options export-minimal --export \
+#     ${fingerprint} >> scripts/operator-sdk-signing-key.asc
+#   (replacing ${fingerprint} with the full fingerprint);
+# * to update scripts/operator-sdk-signing-keyring.gpg, run
+#     gpg --no-options -q --batch --no-default-keyring \
+#     --output scripts/operator-sdk-signing-keyring.gpg \
+#     --dearmor scripts/operator-sdk-signing-key.asc
+$(OPERATOR_SDK):
+	curl -Lo $@ "https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu"
+	curl -Lo $@.asc "https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu.asc"
+	gpgv --keyring scripts/operator-sdk-signing-keyring.gpg $@.asc $@
+	sha256sum -c scripts/operator-sdk.sha256
```

Comment:
Why do we need gpgv + sha256? 

I think gpgv would be enough and we don't need to maintain the hash locally.

Unless I'm missing something of course.

Reply:
The two serve different purposes, and the `gpgv` part is really only an example.

The checksums allow us to verify that the binary we get matches the binary we expect: we don’t have to store the binary ourselves, we can download it as needed, but we do want to ensure that it hasn’t been tampered with.
The GPG key allows us to give some level of trust to unknown artifacts coming from the upstream repository; it’s only really useful when we bump the version. It doesn’t protect us from wilful tampering by someone with access to the corresponding private key.